packages/esm-patient-notes-app/src/types/index.ts

```diff
@@ -32,29 +33,33 @@ export interface PatientNote {
 }
 
 export interface DiagnosisData {
-  word: null;
-  conceptName: {
-    id: number;
+  uuid: string;
+  display: string;
+  name: {
     uuid: string;
     conceptNameType: string;
     name: string;
   };
-  concept: {
-    id: number;
+  conceptClass: {
     uuid: string;
-    conceptMappings: Array<ConceptMapping>;
-    preferredName: string;
+    name: string;
+    description: string;
   };
+  names: Array<conceptNames>;
+  mappings: Array<ConceptMapping>;
+}
+export interface conceptNames {
```

Comment:
Should be capitalised, but again, I don't think this is useful information to maintain...

```suggestion
export interface ConceptNames {
```